src/Microsoft.DotNet.Configurer/CliFallbackFolderPathCalculator.cs

```diff
@@ -1,33 +0,0 @@
-// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.IO;
-using System.Runtime.InteropServices;
-using Microsoft.DotNet.Cli.Utils;
-using Microsoft.DotNet.PlatformAbstractions;
-using NuGet.Common;
-
-namespace Microsoft.DotNet.Configurer
-{
-    public class CliFallbackFolderPathCalculator
```

Comment:
This is a rename, I added more and used it to calculate tools folder path. If you think the rename get too noisy, I'll make a separate PR for it